src/core/core.controller.js

```diff
@@ -529,6 +529,9 @@ module.exports = function(Chart) {
 
 			me.drawDatasets(easingValue);
 
+			// notify before drawing tooltips
+			plugins.notify(me, 'beforeTooltips', [easingValue]);
```

Comment:
Wouldn't `beforeTooltipDraw` be more explicit and thus durable (e.g. if we decide to add `beforeTooltipUpdate` or whatever related to the tooltip)? Also, since there is only one managed tooltip, not sure it should be plural.

Reply:
Good point. I'd agree with this change

Reply:
Sounds good, I'll update it.